packages/wonder-blocks-core/src/hooks/__tests__/use-is-mounted.test.ts

```diff
@@ -1,39 +0,0 @@
-// eslint-disable-next-line import/no-unassigned-import
-import "jest-extended";
-import {renderHook} from "@testing-library/react-hooks";
-
-import {useIsMounted} from "../use-is-mounted";
-
-describe("useIsMounted", () => {
-    it("should return false on first call", () => {
-        // Arrange
-
-        // Act
-        const {result} = renderHook(useIsMounted);
-
-        // Assert
-        expect(result.current()).toBeTrue();
-    });
```

Comment:
**note:** This test case was not doing what was intended. Note that the assertion is different from the test case title; it was mounting the component rather than testing initial render.